templates/index.liquid

```diff
@@ -2,3 +2,7 @@
   {%  section 'slider-collection' %}
   {%  section 'products' %}        
 </div>
+<section class="categories">
+  <h2 class="categories__title">Categorías para ti</h2>
+  {% section 'categories' %}
+</section>
```

Comment:
If you already have a section, why do you wrap the section in another section? Remember that we talked about it: you can assign a tag and a class to the section file.

https://shopify.dev/docs/themes/architecture/sections/section-schema#tag

Also, you should allow the admin to edit the section title.

```suggestion
{% section 'categories' %}
```